dev/tasks/build/start.frag

```diff
@@ -0,0 +1,13 @@
+( function ( root, factory ) {
+	// Register the CKEDITOR global.
+	root.CKEDITOR = factory();
```

Comment:
Why do we pollute the global namespace even though someone wants to use CKEditor5 as an AMD module? That's completely against AMD principles...